runtime/src/lib.rs

```diff
@@ -598,7 +601,10 @@ impl orml_unknown_tokens::Config for Runtime {
 pub struct VirtoDisputeResolver;
 impl virto_payment::DisputeResolver<AccountId> for VirtoDisputeResolver {
 	fn get_origin() -> AccountId {
-		Sudo::key()
+		match Sudo::key() {
```

Comment:
Is this returning an `Option` a new thing? I think it's valid to `expect` that the sudo key is set

Reply:
Yes the sudo key being optional is part of the update of 0.9.16. Changed to expect a sudo key always